### ISSUE TITLE
Bluebird warnings not resolved

### ISSUE THREAD

**Issue by dhm116** [zkat/cipm#66]
It looks like when #58 was resolved by merging #59 and `libcipm` was published to npm as `v3.0.3`, it was actually tagged and published as `v3.0.3` [without the merged fixes from #59](https://github.com/zkat/cipm/blob/v3.0.3/index.js#L351).

The `latest` branch [does indeed include the fixes from #59](https://github.com/zkat/cipm/blob/latest/index.js#L366), however you can also see that the [package.json file was not updated properly to `v3.0.3`](https://github.com/zkat/cipm/blob/latest/package.json).

Would it be possible to re-tag `latest` as `v3.0.4` and republish? At least that way this fix could be included whenever the next `NPM` release is made available.

Thanks!

**Comment by jplock** [comment]
@jehy @zkat any thoughts on this one?

**Comment by jehy** [comment]
Yup, I think that it is an important fix, but I grew a bit tired from pushing this pull request for a long time (October 2018)...

Also it seems that there are new accepted PRs so it would be really good to publish a new version.
